### ISSUE TITLE
Join- Tell us about yourself page Search & Checkout buttons missing, Consultant details not displayed correctly

### ISSUE THREAD

**Issue by monicakarnes** [tastefully-simple/cornerstone#182]
### Expected behavior
Images from Production- Search buttons and Checkout button
![image](https://user-images.githubusercontent.com/57115301/88114551-bbc5b880-cb79-11ea-83ea-3de3261af7ca.png)

Images from Production for Consultant details,and error message alignment 
![image](https://user-images.githubusercontent.com/57115301/88114704-1a8b3200-cb7a-11ea-9a6b-0c2888eb97cb.png)

### Actual behavior

Search only the words - should be Search buttons
![image](https://user-images.githubusercontent.com/57115301/88114487-90db6480-cb79-11ea-891f-4cf42fa128d2.png)

Find your Sponsor- Consultants info is all spread out-  and huge icons for email and web-page-  See above production image for details 
![image](https://user-images.githubusercontent.com/57115301/88114098-c3d12880-cb78-11ea-99dc-520b36e95d6b.png)

Checkout arrow- should be a  teal Checkout button 
![image](https://user-images.githubusercontent.com/57115301/88114189-efeca980-cb78-11ea-805d-f587f42c4d58.png)

Error messages misplaced due to checkout arrow - needs alignment 
![image](https://user-images.githubusercontent.com/57115301/88114259-16124980-cb79-11ea-8b9e-ea06ccd63aec.png)

### Steps to reproduce behavior
Go to https://tastefully-simple-sandbox-2.mybigcommerce.com/join-us
 Click on Join Now! Link
 Sign up
 enter in new Join details and click to get to Whats in your kit page
click next to get to the Tell us about yourself page
complete details and click check out

**Comment by staciejurczak** [comment]
Per External Slack with Jared noting that this error is a result/broke when the Irish titan CSS files were removed.
- confirm if we know which CSS files for Razoyo to add back in.

**Comment by TastefullySimple** [comment]
I'm not sure we want to add those files back. I think it would be better to use styles from the style guide. Below are the mock-up the form was based on. Please note the consultant results list styling should match current consultant results styling and not what is in the 5th picture.

![join-page-1-sign-in](https://user-images.githubusercontent.com/53190431/88309809-7b2e8200-ccd4-11ea-838d-4b0b222fe84d.png)
![join-page-1-sign-up](https://user-images.githubusercontent.com/53190431/88309817-7d90dc00-ccd4-11ea-91ea-eaffd9b26183.png)
![join-page-2](https://user-images.githubusercontent.com/53190431/88309823-808bcc80-ccd4-11ea-9990-34c32795d559.png)
![join-page-3](https://user-images.githubusercontent.com/53190431/88309826-82559000-ccd4-11ea-9e73-b76ae3c75da9.png)
![join-page-3-consultant-sreach](https://user-images.githubusercontent.com/53190431/88309830-841f5380-ccd4-11ea-911b-bc665b2c2f79.png)

**Comment by staciejurczak** [comment]
https://razoyo.atlassian.net/browse/TST-132

**Comment by wbyrnetx** [comment]
Pull request is in #205, waiting for it to be deployed.

**Comment by monicakarnes** [comment]
This still needs to get fixed, for the sponsor - consultant information - here is what I see
![image](https://user-images.githubusercontent.com/57115301/88831354-d78e1780-d194-11ea-9cf2-c23122c050d9.png)

here is what we should see for the sponsor- consultant information - aka currently from production- 
![image](https://user-images.githubusercontent.com/57115301/88831553-1c19b300-d195-11ea-9dfa-c4f7e8f1678e.png)

**Comment by wbyrnetx** [comment]
This passed **most** of our internal QA. But @monicakarnes , the last thing you brought up we are still working on. We created a separate ticket for it so that another dev could work in parallel to Jen [TST-132](https://razoyo.atlassian.net/browse/TST-132). It has to do with the fact that this page's styles conflict with the style guide styles a little bit, as it was created before we had made the style guide.

Once [TST-132](https://razoyo.atlassian.net/browse/TST-132) is done, then I will let you know and QA should pass on all the items mentioned in here.

**Comment by wbyrnetx** [comment]
@monicakarnes  can you check it again? I just tested on the Tech Sandbox and this is what I see when I do a consultant search for "Julie" in Minnesota
<img width="1356" alt="Screen Shot 2020-08-11 at 11 27 22 AM" src="https://user-images.githubusercontent.com/4029452/89916626-d5c94880-dbc5-11ea-8d8e-64d3c1b64046.png">

**Comment by monicakarnes** [comment]
![image](https://user-images.githubusercontent.com/57115301/89942193-bb9d6380-dbe1-11ea-8357-7fc1d731a1f1.png)

**Comment by wbyrnetx** [comment]
Jared says to test in QA1 to fix the images.

**Comment by monicakarnes** [comment]
consultant details, are now displaying correctly- closing this, will check back to make sure images load upon fix of bug 253
